docs/rpyc.rst

```diff
@@ -9,14 +9,24 @@ processes and computers, so that remote objects can be manipulated as if they
 were local. Here are simple steps you need to follow in order to install and
 use RPyC with ev3dev:
 
-1. Install RPyC both on the EV3 and on your desktop PC with the following
-   command (depending on your operating system, you may need to use ``pip3`` or
-   ``pip`` on the PC instead of ``easy_install3``):
+1. Install RPyC both on the EV3 and on your desktop PC. On the EV3 the following
+   command should work:
 
    .. code-block:: shell
 
        sudo easy_install3 rpyc
 
+   On the desktop PC, it really depends on your operating system. In case its
```

Comment:
I think here 'its' is a short form of 'it is', so no apostrophe? But I guess I should change it to 'it is' anyway.